modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,317 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-5_{context}"]
+= Release notes for {pipelines-title} General Availability 1.5
+
+{pipelines-title} General Availability (GA) 1.5 is now available on {product-title} 4.8.
+
+[id="compatibility-support-matrix-1-5_{context}"]
+== Compatibility and support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Compatibility and support matrix
+[cols="1,1,1",options="header"]
+|===
+| Feature | Version | Support Status
+| Pipelines | 0.24 | GA
+| CLI | 0.19 | GA
+| Catalog | 0.24 | GA
+| Triggers | 0.14 | TP
+| Pipeline resources | - | TP
+|===
+
+
+For questions and feedback, you can send an email to the product team at pipelines-interest@redhat.com.
+
+[id="new-features-1-5_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.5.
+
+* Pipeline run and task runs will be automatically pruned by a cron job in the target namespace. The cron job uses the `IMAGE_JOB_PRUNER_TKN` environment variable to get the value of `tkn image`. With this enhancement, the following fields are introduced to the `TektonConfig` custom resource:
```

Comment:
IBM SG, page 68. We can use future tense, but it's often recommended to avoid. Is it more appropriate and common to use future tense in release notes? 

If so, don't concern yourself with the future tense here. However, if you do want to change it to present tense, consider,
`The cron job in the target namespace automatically prunes the pipeline run and tasks. The cron job uses the...` 

I think this rewrite is more consistent with the construction of your second sentence.

Reply:
It is not appropriate to make the cron job as subject in the sentence, as the emphasis is on pipeline run and task run.